mix.exs

```diff
@@ -0,0 +1,39 @@
+defmodule HWApp.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :hw_app,
+      version: "0.1.0",
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      dialyzer: [
+        plt_add_apps: [:mix, :ex_unit, :iex],
+        ignore_warnings: ".dialyzer-ignore.exs",
+        list_unused_filters: true,
+        plt_file: {:no_warn, "priv/plts/dialyzer.plt"}
+      ],
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        coveralls: :test,
+        "coveralls.detail": :test
+      ]
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  defp deps do
+    [
+      {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
+      {:credo_contrib, "~> 0.2.0", only: [:dev, :test], runtime: false},
```

Comment:
Literally all known to me _Elixir_ libraries use semantic versioning, hence
```suggestion
      {:credo_contrib, "~> 0.2", only: [:dev, :test], runtime: false},
```

Reply:
I think both examples use semantic versioning, however I haven't noticed that deps consist of major and minor versions without patch one. This is true. Thanks!